modules/andes-core/broker/src/main/java/org/wso2/andes/configuration/enums/AndesConfiguration.java

```diff
@@ -59,6 +59,20 @@
      */
     COORDINATION_THRIFT_SERVER_PORT
             ("coordination/thriftServerPort", "7611", Integer.class),
+
+    /**
+     * The IP of the Thrift server to be advertised to external. This is used in NAT/containerized
+     * environments which has a local private IP and a routable public IP.
+     */
+    COORDINATION_THRIFT_SERVER_ADVERTISED_HOST
+            ("coordination/thriftServerAdvertisedHost", "", String.class),
+
+    /**
+     * The port of the Thrift server to be advertised to external. This is used in NAT/containerized
+     * environments which has a local private IP and a routable public IP.
+     */
+    COORDINATION_THRIFT_SERVER_ADVERTISED_PORT
+            ("coordination/thriftServerAdvertisedPort", "-1", Integer.class),
             
```

Comment:
Again, this is not a mandatory property. If we set it to a default value, AFAIU we won't be able to distinguish if the user wants to use this property or not.